searchcore/src/vespa/searchcore/proton/server/transactionlogmanagerbase.h

```diff
@@ -5,25 +5,27 @@
 #include <vespa/searchlib/transactionlog/translogclient.h>
 #include <mutex>
 #include <condition_variable>
+#include <vespa/fastos/timestamp.h>
 
 namespace proton {
 
 /**
  * Base class managing the initialization and replay of a transaction log.
  **/
 class TransactionLogManagerBase {
-
-    search::transactionlog::TransLogClient _tlc;
-    search::transactionlog::TransLogClient::Session::UP _tlcSession;
-    vespalib::string            _domainName;
-    mutable std::mutex          _replayLock;
+protected:
+    using TransLogClient = search::transactionlog::TransLogClient;
+private:
+    TransLogClient                  _tlc;
+    TransLogClient::Session::UP     _tlcSession;
+    vespalib::string                _domainName;
+    mutable std::mutex              _replayLock;
     mutable std::condition_variable _replayCond;
-    volatile bool               _replayDone;
-    bool                        _replayStarted;
-    double                      _replayStartTime;
+    volatile bool                   _replayDone;
+    bool                            _replayStarted;
+    mutable fastos::StopWatch       _replayStopWatch;
```

Comment:
prefer not using mutable when not needed

Reply:
Fixed